gremlin-client/src/structure/value.rs

```diff
@@ -339,6 +339,7 @@ impl std::convert::TryFrom<GValue> for uuid::Uuid {
     fn try_from(value: GValue) -> GremlinResult<Self> {
         match value {
             GValue::Uuid(uid) => Ok(uid),
+            GValue::List(s) => from_list(s),
```

Comment:
Having to add these GValue::List instances kind of surprised me. Seems like if we're being sent back a List element something should be wrong? But from stepping through with a debugger we end up here from the `impl_try_from_option` macro which means it's the GValue coming from Tinkerpop 🤔

Reply:
Stepping through with the debugger a little earlier, basically before it creates the properties Map I then deserialize values out of it seems to flow from deserialize_map to deserialize_list to deserialize_XXX where XXX are the various deserialize methods in https://github.com/wolf4ood/gremlin-rs/blob/master/gremlin-client/src/io/serializer_v3.rs does that seem correct / working as intended @wolf4ood ?

Reply:
Yeah it' seems strange that you had to add the list variant let me check :)

Reply:
Probably it has something to do with multivalue properties but let me recall in my head. The try from stuff was added for doing automatic structs binding :)

Reply:
It seemed like it was creating singleton lists in the debugger when I was walking through the steps and then unpacking the single item out of the list. I'm working on #127 now so I may have more insight as make my way through it.

Reply:
If I remember correctly when doing value map step the map  contains the values in form of array for supporting multi-properties on vertex type